tests/hap/test_align.py

```diff
@@ -8,148 +8,204 @@
+            output=False,
+        )
 
         # Examine the output table to extract the RMS for the entire fit and the compromised
         # information
         if dataset_table:
-            total_rms = dataset_table.filtered_table['total_rms'][0]
+            total_rms = dataset_table.filtered_table["total_rms"][0]
 
         assert 0.0 < total_rms <= RMS_LIMIT
 
-
-    @pytest.mark.parametrize("input_filenames", [['j8ura1j1q_flt.fits', 'j8ura1j2q_flt.fits',
-                                                  'j8ura1j4q_flt.fits', 'j8ura1j6q_flt.fits',
-                                                  'j8ura1j7q_flt.fits', 'j8ura1j8q_flt.fits',
-                                                  'j8ura1j9q_flt.fits', 'j8ura1jaq_flt.fits',
-                                                  'j8ura1jfq_flt.fits', 'j8ura1jgq_flt.fits',
-                                                  'j8ura1jhq_flt.fits', 'j8ura1jiq_flt.fits',
-                                                  'j8ura1jjq_flt.fits', 'j8ura1jkq_flt.fits'],
-                                                 ['j92c01b4q_flc.fits', 'j92c01b5q_flc.fits',
-                                                  'j92c01b7q_flc.fits', 'j92c01b9q_flc.fits'],
-                                                 ['jbqf02gzq_flc.fits', 'jbqf02h5q_flc.fits',
-                                                  'jbqf02h7q_flc.fits', 'jbqf02hdq_flc.fits',
-                                                  'jbqf02hjq_flc.fits', 'jbqf02hoq_flc.fits',
-                                                  'jbqf02hqq_flc.fits', 'jbqf02hxq_flc.fits',
-                                                  'jbqf02i3q_flc.fits', 'jbqf02i8q_flc.fits',
-                                                  'jbqf02iaq_flc.fits'],
-                                                 ['ib2u12kaq_flt.fits', 'ib2u12keq_flt.fits',
-                                                  'ib2u12kiq_flt.fits', 'ib2u12klq_flt.fits'],
-                                                 ['ibnh02coq_flc.fits', 'ibnh02cmq_flc.fits',
-                                                  'ibnh02c7q_flc.fits', 'ibnh02c5q_flc.fits',
-                                                  'ibnh02cpq_flc.fits', 'ibnh02c9q_flc.fits',
-                                                  'ibnh02bfq_flc.fits', 'ibnh02beq_flc.fits']])
+    @pytest.mark.parametrize(
+        "input_filenames",
+        [
+            [
+                "j8ura1j1q_flt.fits",
+                "j8ura1j2q_flt.fits",
+                "j8ura1j4q_flt.fits",
+                "j8ura1j6q_flt.fits",
+                "j8ura1j7q_flt.fits",
+                "j8ura1j8q_flt.fits",
+                "j8ura1j9q_flt.fits",
+                "j8ura1jaq_flt.fits",
+                "j8ura1jfq_flt.fits",
+                "j8ura1jgq_flt.fits",
+                "j8ura1jhq_flt.fits",
+                "j8ura1jiq_flt.fits",
+                "j8ura1jjq_flt.fits",
+                "j8ura1jkq_flt.fits",
+            ],
```

Comment:
Recall we agree to remove this dataset from the parameterized list so all the tests in this file will pass.  Then yo were going to create two new tickets: (1) process the entire dataset as a single visit, and (2) process the single "bad" file.